From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: PJ Reiniger <pj.reiniger@gmail.com>
Date: Fri, 20 Oct 2023 16:40:59 -0400
Subject: [PATCH 11/11] Fix bazel

---
 src/win/util.c | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/src/win/util.c b/src/win/util.c
index 4b76417fcbac2480725471740c037deb859e17ca..ecf31ad800fde781b3eabf09ebf5fd27a95d31ee 100644
--- a/src/win/util.c
+++ b/src/win/util.c
@@ -19,6 +19,11 @@
  * IN THE SOFTWARE.
  */
 
+// Bazel hardcodes a single version of windows, which does not have all the functions we need.
+#ifdef _WIN32_WINNT
+#undef _WIN32_WINNT
+#endif
+
 #include <assert.h>
 #include <direct.h>
 #include <limits.h>
